pact/ns.pact

```diff
@@ -1,5 +1,8 @@
+(define-keyset 'swap-ns-user)
+(define-keyset 'swap-ns-admin)
+(ns.write-registry 'swap (keyset-ref-guard 'swap-ns-admin) true)
```

Comment:
This will break in `exchange.repl` tests no?

Reply:
Added ns.pact and loaded in the exchange.repl.